### PR TITLE
Content box

### PR DESCRIPTION
**Pull request by frideska**
# PR Checklist

## What type of change is this?
- [ ] Internal (Does not need to be published)
- [x] Patch (No new feature and no breaking change)
- [ ] Minor (New feature, but non breaking)
- [ ] Major (Breaking change)

## Have you bumped the package version?
- [x] Yes
- [ ] No, does not need to be published

## Does this PR relate to an open issue?
- [x] Yes (remember to link)
- [ ] No

## Describe PR briefly:
* Removed all box-sizing: content-box

https://trello.com/c/rKYhg3g8/396-fiks-alle-tilfeller-av-content-box

### CODE REVIEW

File: components/form/radiobutton.scss [hunk 1]
```diff
@@ -9,7 +9,6 @@ $form-radiobutton-mark-border: 1px solid var(--hdd-color-primary);
   display: flex;
   align-items: center;
   cursor: pointer;
-  box-sizing: content-box;
```
Comment (JGH153) [inline components/form/radiobutton.scss]: image

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
+# Update [9.0.0]
+In this update the size of `.hdd-form_checkbox`, `.hdd-form_checkbox-toggle` and `.hdd-form_radiobutton` has increased from 16px to 18px.
+
```
Comment (JGH153) [inline CHANGELOG.md]: Hvorfor ikke holde på 16?

Comment (frideska) [reply CHANGELOG.md]: var bare sånn det var i designet

Comment (frideska) [reply CHANGELOG.md]: Kan spørre hva design synes om 16px vs 18px i morgen :)

Comment (frideska) [reply CHANGELOG.md]: Endret til orginale størrelser, utenom selve sirkelen i checkbox-toggle er pittelitt større, så vil ikke si det er breaking

**Pull request merged**